src/wizard/index.tsx

```diff
@@ -89,9 +88,10 @@ export default function Wizard({
     );
   }
 
-  useEffectOnUpdate(() => {
+  useEffect(() => {
```

Comment:
Why replacing useEffectOnUpdate with useEffect?

Reply:
useEffectOnUpdate skips the initial render as intended and so  then I never got a metric on the 1st render 😅